db/data/sample/groups.yml

```diff
@@ -0,0 +1,20 @@
+---
```

Comment:
"automatic" was the wrong word.

The idea is that three sets of data would be created initially. The 'Federal', 'Health', and 'generic' sets. These datasets should be able to be installed without a rake task.

Ideally, the UI used to installed them can inform the admin user if a dataset has been installed and which one, and also provide a way to uninstall the dataset (pie in the sky stretch).